### ISSUE TITLE
User can not provide review interests during registration

### ISSUE THREAD

**Issue by NateWr** [NateWr/bootstrap3#171]
In most themes, when a user indicates that they are interested in reviewing for the journal, they are shown a field where they can enter their reviewer interests. This is not shown in the Bootstrap3 theme.

See the [original report](https://forum.pkp.sfu.ca/t/bootstrap-3-reviewer-interest-input-field-missing/68373/8) on the forum.

**Comment by asmecher** [comment]
@kaitlinnewson, thanks, this looks good -- I assume these snippets are taken from their default theme equivalent templates and lightly adapted? Please go ahead and merge, and when forward-porting, it's probably best to go back to the source templates and make sure they haven't changed between releases.

**Comment by kaitlinnewson** [comment]
@asmecher Yes they are based on the default theme. I'll be sure to have a look at those when forward-porting.

**Comment by kaitlinnewson** [comment]
Completed in all 3 branches, closing.
